roles/openshift_health_checker/library/search_journalctl.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/python
+"""Interface to journalctl"""
+
+from time import time
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+def _exit_json(module, errors, matched_expressions, msg):
+    module.exit_json(
+        changed=False,
+        failed=len(errors),
+        errors=errors,
+        matched=matched_expressions,
+        msg=msg,
+    )
+
+
+def _stamp_too_old(stamp, time_limit):
+    epoch = int(stamp) / 1000000
+    return epoch - (time() - time_limit) < 0
```

Comment:
Yeah, loosely based this on its oadm diagnostics counterpart https://github.com/openshift/origin/blob/master/pkg/diagnostics/systemd/analyze_logs.go#L142